### ISSUE TITLE
Is there a requirement for the node version?

### ISSUE THREAD

**Issue by sundyxfan** [sahat/hackathon-starter#345]
```
$ node app.js
```
~~报错~~ Error

```
const Promise = require('bluebird');
^^^^^
SyntaxError: Use of const in strict mode.
```

```
$ node --harmony app.js
```
~~报错~~ Error
```
 class MongoStore extends Store {
    ^^^^^
SyntaxError: Unexpected reserved word
```

And, switch to node version to v.5.0.0, some modules can not be used

why？

**Comment by sahat** [comment]
I am not using ES6 syntax anywhere in the code. Did you do those changes yourself?

**Comment by sahat** [comment]
Ah I see, connect-mongo is using the new ES6 syntax. I will try to fix it.

**Comment by sahat** [comment]
<img width="840" alt="screenshot 2015-12-25 17 14 01" src="https://cloud.githubusercontent.com/assets/544954/12004483/ed32a8f2-ab2a-11e5-829b-5bbf4eb1a18a.png">

**Comment by sahat** [comment]
Fixed in 0cf273dac812e0f1edf5f251eb06dfa3f264a1e1
